nemo/src/io/format_builder.rs

```diff
@@ -231,7 +231,9 @@ impl<Tag> FormatParameter<Tag> for StandardParameter {
 
         match self {
             StandardParameter::BaseParamType(no_parameters) => match *no_parameters {},
-            StandardParameter::Resource => Ok(()),
+            StandardParameter::Resource => ResourceBuilder::try_from(value)
+                .map(|_| ())
```

Comment:
we're using `.and(Ok(()))` elsewhere, would be nice to stay consistent (also several times below)